Assets/Scripts/UIManager.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+
+public class UIManager : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+
+ 
+    public TextMeshProUGUI playerOneScoreLabel;
+    public TextMeshProUGUI playerOneLivesLeftLabel;
+    public GameObject playerOneDurability;
+    public GameObject playerOneHealth;
+    public GameObject playerOneInventory;
+
+
+    public int playerOneScoreNum;
```

Comment:
You shouldn't make 4 variables for each player. You should make a class that represents a player's UI status and make 4 of that class for each player.